website/docs/api/language.md

```diff
@@ -63,18 +63,18 @@ spaCy loads a model under the hood based on its
-| **RETURNS**                           | The initialized object. ~~Language~~                                                                                                                                                                                                                                 |
+| Name                                  | Description                                                                                                                                                                                                                                                                                                        |
+| ------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `config`                              | The loaded config. ~~Union[Dict[str, Any], Config]~~                                                                                                                                                                                                                                                               |
+| _keyword-only_                        |                                                                                                                                                                                                                                                                                                                    |
+| `vocab`                               | A `Vocab` object. If `True`, a vocab is created using the default language data settings. ~~Vocab~~                                                                                                                                                                                                                |
+| `disable`                             | Name(s) of pipeline component(s) to [disable](/usage/processing-pipelines#disabling). Disabled pipes will be loaded but they won't be run unless you explicitly enable them by calling [nlp.enable_pipe](/api/language#enable_pipe). Is merged with the config entry `nlp.disabled`. ~~Union[str, Iterable[str]]~~ |
+| `enable` <Tag variant="new">3.4</Tag> | Name(s) of pipeline component(s) to [enable](/usage/processing-pipelines#disabling). All other pipes will be disabled, but can be enabled again using [nlp.enable_pipe](/api/language#enable_pipe). Overwrites the config entry `nlp.enabled`. ~~Union[str, Iterable[str]]~~                                       |
```

Comment:
There is no `nlp.enabled` config entry, there's just `nlp.pipeline` and `nlp.disabled`.

Reply:
Ah, indeed. Why not?

Reply:
It would be redundant with `nlp.pipeline - nlp.disabled`.

Reply:
Right, but that also applies to the `enable`/`disable` situation? Anyway, not important - thanks for the catching this.